runtime/kusama/src/constants.rs

```diff
@@ -18,30 +18,22 @@
 pub mod currency {
 	use primitives::v0::Balance;
 
-	pub const DOTS: Balance = 1_000_000_000_000;
-	pub const DOLLARS: Balance = DOTS / 300;
-	pub const CENTS: Balance = DOLLARS / 100;
+	pub const UNITS: Balance = 1_000_000_000_000;
```

Comment:
Removed tehe questionable `DOLLARS` to leave the (slightly more vague) `CENTS` and `GRAND`.

Reply:
Not everywhere though? (still in Polkadot, Rococo, and Test-Runtime)